samples/boards/nrf52/sensor_temperature/src/main.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <device.h>
+#include <gpio.h>
+#include <sensor.h>
+
+#define LED_PORT	DT_GPIO_LEDS_LED0_GPIO_CONTROLLER
+#define LED0		DT_GPIO_LEDS_LED0_GPIO_PIN
+#define LED1		DT_GPIO_LEDS_LED1_GPIO_PIN
+#define LED2		DT_GPIO_LEDS_LED2_GPIO_PIN
+#define LED3		DT_GPIO_LEDS_LED3_GPIO_PIN
+int led_write_values[4] = {0};
+
+/* 1000 msec = 1 sec */
+#define SLEEP_TIME	1000
+
+#define TEMPERATURE_READ_FAILED  1
```

Comment:
Do we need to define such label? Wouldn't it more adequate to return the value returned by sensor API functions (or even `-EIO`)?

Reply:
No, we don't. I will remove it and use the return value of the sensor API.